modules/OAuth2Clients/language/en_us.lang.php

```diff
@@ -48,6 +48,7 @@
     'LBL_MODULE_ID'=> 'OAuth Clients',
     'LBL_IS_CONFIDENTIAL'=> 'Is confidential',
     'LBL_SECRET'=> 'Secret',
+    'LBL_SECRET_HASHED'=> 'Secret (This value will not be accessible once you press save. Make sure to remember it.)',
```

Comment:
You may want to change the " Make sure to remember it". Someone may take this as being to harsh.

What about ?   "Secret (Please take a note of the secret as it will not be available after you save)"

Also can we move the warning below the text field? Perhaps we should make this